resources/govuk-template.mustache.html

```diff
@@ -158,19 +158,47 @@
         <header role="banner" id="global-header" class="with-proposition">
             <div class="header-wrapper">
 
-                <div class="header-global">
-                    <div class="header-logo">
-                        <a href="https://www.gov.uk/" title="Go to the GOV.UK homepage" data-journey-click="header:Click:GOV UK" id="logo" class="content">
-                            <img src="/template/assets/images/gov.uk_logotype_crown.png" alt=""> GOV.UK
-                        </a>
-                    </div>
+                {{#newGovukLogo}}
+                <div class="govuk-header__logo">
+                    <a href="https://www.gov.uk" class="govuk-header__link govuk-header__link--homepage">
+                        <span class="govuk-header__logotype">
+                          <!--[if gt IE 8]><!-->
```

Comment:
do we need this/does hmrc still support IE8?

Reply:
https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices

Reply:
Yes you are right @Luka-Abey but I coped it as it was in design system https://design-system.service.gov.uk/components/header/ :)

Reply:
fantastic, thanks